i18n/en.json

```diff
@@ -1,15 +1,15 @@
 {
   "script-name": "Convenient Discussions",
-  "script-name-genitive": "{{int:convenientdiscussions-script-name}}",
-  "script-name-prepositional": "{{int:convenientdiscussions-script-name}}",
+  "script-name-genitive": "Convenient Discussions",
+  "script-name-prepositional": "Convenient Discussions",
   "script-name-short": "CD",
 
-  "user-male-dative": "",
-  "user-male-genitive": "",
-  "user-female-dative": "",
-  "user-female-genitive": "",
-  "user-unknown-dative": "",
-  "user-unknown-genitive": "",
+  "user-male-dative": "user",
+  "user-male-genitive": "user",
+  "user-female-dative": "user",
+  "user-female-genitive": "user",
+  "user-unknown-dative": "user",
+  "user-unknown-genitive": "user",
```

Comment:
This shouldn't be changed. It should be filled only if the phrase "reply to <Username>" sounds silly in the language (usually, because the username lacks inflection) and "reply to user <Username>" is needed.